src/html.rs

```diff
@@ -89,4 +121,49 @@ mod test {
     fn static_config_parses() {
         assert!(!CONFIG.sections_to_remove.is_empty());
     }
+
+    #[test]
+    fn remove_links() {
+        let html = r#"
+        <p> Some text that includes
+            <a href="Some_Page"><span id="inner-content">several</span></a>
+            <a id="second-link" href="./Another_Page">relative links</a>
+        and
+            <a href="https://example.com/page">an absolute link</a>
+        .
+        </p>
+        "#;
+
+        let anchors = Selector::parse("a").unwrap();
+        let inner_element = Selector::parse("#inner-content").unwrap();
+        let second_link = Selector::parse("#second-link").unwrap();
+
+        let mut document = Html::parse_fragment(html);
+        let links: Vec<_> = document
+            .select(&anchors)
+            .filter_map(|el| el.value().attr("href"))
+            .collect();
+
+        eprintln!("{}", document.html());
+
+        assert_eq!(
+            vec!["Some_Page", "./Another_Page", "https://example.com/page"],
+            links
+        );
+
+        // orphan one of the links
+        let link = document.select(&second_link).next().unwrap().id();
+        document.tree.get_mut(link).unwrap().detach();
+
+        super::remove_links(&mut document);
+
+        let links: Vec<_> = document.select(&anchors).collect();
+
+        assert!(links.is_empty(), "all links should be removed");
```

Comment:
nit: Using English sentences starting with a capital letter and ending with a dot may be more readable.